apps/src/templates/projects/ProjectWidget.jsx

```diff
@@ -10,6 +13,46 @@ const ProjectWidget = React.createClass({
     projectList: PropTypes.arrayOf(personalProjectDataPropType).isRequired
   },
 
+  statics: {
+    setupProjectWidget() {
+      $.ajax({
+        method: 'GET',
+        url: `/v3/channels`,
+        dataType: 'json'
+      }).done(projectLists => {
+        const widget = document.getElementById('projects-widget');
+        projectLists.sort((a, b) => {
+          if (a.updatedAt < b.updatedAt) {
+            return 1;
+          } else {
+            return -1;
+          }
+        });
+        projectLists = projectLists.filter(project => {
+          return !project.hidden;
+        });
+        projectLists = projectLists.slice(0,4);
+        let convertedProjectList = [];
+        for (let i = 0; i < projectLists.length; i++){
```

Comment:
Since our project cards use can render data from different APIs, the data is formatted in different ways. So depending on the API call, the data needs to be mapped/filtered a bit. I didn't add this to the component, but had this as an example of how I tested the component when I built it. But yes, comment would be great.